hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -944,8 +944,11 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status,
     OmKeyInfo keyInfo = status.getKeyInfo();
     short replication = (short) keyInfo.getReplicationConfig()
         .getRequiredNodes();
+    //calculate the number of bytes required to store the dataSize with replication
+    long diskConsumed = keyInfo.getReplicatedSize();
```

Comment:
Same as above, I think it would be cleaner to just use the `keyInfo.getReplicatedSize()` call inline.

Reply:
yes!!